check/test.go

```diff
@@ -232,7 +236,22 @@ func (t testItem) evaluate(s string) *testOutput {
 	}
 
 	result.flagFound = match
-	glog.V(3).Info(fmt.Sprintf("found %v", result.flagFound))
+	var isExist string
+	if !result.flagFound{
+		isExist = " not"
+	}
```

Comment:
Personally I would just trace out "exists: true" or "exists: false" so that you can do it by formatting the boolean value of `result.flagFound`. 

But if you want to make it more of a sentence, for grammatical accuracy it's "X exists" or "X does not exist".  So you could do something like 

```suggestion
	var isExist = "exists"
	if !result.flagFound{
		isExist = "does not exist"
	}
```

and then drop the word "exist" from the format strings later